fable/conanfile.py

```diff
@@ -53,11 +55,22 @@ def build_requirements(self):
 
     def layout(self):
         cmake.cmake_layout(self)
+        self.cpp.source.includedirs.append(os.path.join(self.folders.build, "include"))
```

Comment:
out of curiosity, how is `self.folders.build` different from `self.build_folder`? 
minor nitpick: afaik pathlib is recommended over `os.path`

Reply:
`self.folders.build` vs `self.build_folder`:
Conan is annoying in that not every variable is available in every hook.
I know for a fact that `self.folders.build` is available because I saw it in the Conan source that was used for defining layouts. I'd have to run tests and try out `self.build_folder` to see if that works too.

Reply:
Visa `os.path`: A quick search in our Conanfiles shows >20 instances of `os.path`. It'd be best to replace them all in one swoop, I think.

Unfortunately, not everything we use `os.path` works with `pathlib` :-(
And I tried to not use both in the same Conanfile to keep imports down.